src/harmony/Alert/Alert.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode } from 'react';
+import cn from 'classnames';
+
+import { Text } from '../../harmony/Text/Text';
+
+import s from './Alert.module.css';
+// import { IconExclamationCircleOutline, IconInfoCircleOutline } from '@taskany/icons';
+
+interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
+    view: 'default' | 'danger' | 'warning';
+    text: string;
+    icon: ReactNode;
+}
+const viewMap = {
+    default: s.Alert_default,
+    danger: s.Alert_danger,
+    warning: s.Alert_warning,
+};
+
+const TextColorMap = {
```

Comment:
Do u really need this one? Why .Alert style is not enough?

Reply:
fixed